modules/pool/subgraph-mapper.ts

```diff
@@ -266,24 +266,24 @@ const mapPoolTypeVersion = (poolType: string, poolTypeVersion: number): number =
 };
 
 const dataMapper = {
-    ELEMENT: dataMappers.element,
-    FX: dataMappers.fx,
-    GYRO: dataMappers.gyro,
-    GYRO3: dataMappers.gyro,
-    GYROE: dataMappers.gyro,
-    LINEAR: dataMappers.linear,
+    ELEMENT: element,
+    FX: fx,
+    GYRO: gyro,
+    GYRO3: gyro,
+    GYROE: gyro,
+    LINEAR: linear,
 };
 
 const dynamicMapper = {
-    STABLE: dataMappers.stableDynamic,
-    COMPOSABLE_STABLE: dataMappers.stableDynamic,
-    META_STABLE: dataMappers.stableDynamic,
-    LINEAR: dataMappers.linearDynamic,
+    STABLE: stableDynamic,
+    COMPOSABLE_STABLE: stableDynamic,
+    META_STABLE: stableDynamic,
+    LINEAR: linearDynamic,
```

Comment:
can we remove this? it's unused afaik.

Reply:
mapper is subgraph schema related, so more data we parse to DB schema the better. think about it as a layer. it makes it easy to compose what is needed for any pool syncing we would want to do.

we still read these data from subgraph in pool create jobs, once removed where would it go? just keep it blank until onchain calls fill it in?